deploy/helm/charts/templates/NOTES.txt

```diff
@@ -0,0 +1,5 @@
+The OpenEBS ZFS LocalPV has been installed. Check its status by running:
```

Comment:
once OpenEBS ZFS LocalPV has been installed check its status by running.

Reply:
This text is displayed on the terminal at the end when the helm install is completed.